databases/week1/database1-week1.txt

```diff
@@ -0,0 +1,56 @@
+SELECT COUNT(*)
+FROM task;
+
+SELECT COUNT(*)
+FROM task
+WHERE due_date IS NULL;
+
+SELECT *
+FROM task
+WHERE status_id = (
+		SELECT id
+		FROM STATUS
+		WHERE NAME = 'done'
+		);
+
+SELECT *
+FROM task
+WHERE status_id != (
+		SELECT id
+		FROM STATUS
+		WHERE NAME = 'done'
+		);
```

Comment:
Here I can't see in which status a task is as I (any external viewer) will not understand (not easily if at least) what status_id=3 meant. We don't even have a status name in the table task. So we need to get some information from other related tables (by joining).
Check this if it can help---

```
SELECT task.id,title,due_date, status.name
FROM task
JOIN status ON task.status_id=status.id
WHERE status.name="Done";
```
-- Find all the tasks that are not marked as not done

```
SELECT task.id,title,due_date, status.name
FROM task
JOIN status ON task.status_id=status.id
WHERE status.name!="Done";
```